Framework/Kernel/src/ErrorReporter.cpp

```diff
@@ -96,9 +96,6 @@ std::string ErrorReporter::generateErrorMessage() const {
   message["osVersion"] = ConfigService::Instance().getOSVersion();
   message["osReadable"] = ConfigService::Instance().getOSVersionReadable();
 
-  // legacy interface requires paraview version
```

Comment:
It's quite likely we'll need to keep this as the web api might require the payload to contain it. We should check first.

Reply:
Thank you for letting me know. I will add it back :+1: